runtime/tests/automl_runtime/forecast/deepar/model_test.py

```diff
@@ -145,10 +149,23 @@ def test_model_save_and_load_multi_series(self):
             mlflow_deepar_log_model(deepar_model, sample_input)
 
         run_id = run.info.run_id
-        loaded_model = mlflow.pyfunc.load_model(f"runs:/{run_id}/model")
 
+        # check if all additional dependencies are logged
+        self._check_requirements(run_id)
+
+        # load the model and predict
+        loaded_model = mlflow.pyfunc.load_model(f"runs:/{run_id}/model")
         pred_df = loaded_model.predict(sample_input)
 
         assert pred_df.columns.tolist() == [time_col, "yhat", id_col]
         assert len(pred_df) == self.prediction_length * 2
         assert pred_df[time_col].min() > sample_input[time_col].max()
+
+    def _check_requirements(self, run_id: str):
```

Comment:
nit: if a method is being tested in a TestClass, make it a public method (i.e. without prefix _)

Reply:
It's not a unit test itself, it's a helper method called only within other unit test cases, should I still remove prefix?